### PR TITLE
Markdown WYSIWYG Editor

### PR DESCRIPTION
**Pull request by wmnnd**
This MR doesn’t actually include any WYSIWYG-related code yet, just a refactoring of the inclusion of SVG icons so far.

### CODE REVIEW

**Review by Jcambass — APPROVED** [review]
Very nice! Was actually struggling with icons when I attempted to build the send campaign later UI since it wasn't clear which icon set Keila uses. That's solved now :)

**Comment by wmnnd** [comment]
Here’s what the editor should look like:
Screenshot_2021-03-06 Open Source Newsletter Tool · Keila


I’d be very happy to get some feedback :smile:

File: assets/js/campaign-editor/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                editorView.dom.dispatchEvent(new CustomEvent("x-sync", {
+                    bubbles: true,
+                    detail: { skipDispatch: true }
+                }))
+            }
+        }
+    }
+})
+
+/** Class representing the ProseMirror Markdoen editor used by Keila.
```
Comment (Jcambass) [inline assets/js/campaign-editor/index.js:25]:
```suggestion
/** Class representing the ProseMirror Markdown editor used by Keila.
```

**Review by Jcambass — APPROVED** [review]
Looking good! I can only make limited statements about the JS but nothing weird looking and the end result looks fantastic!

**Comment by Jcambass** [comment]
When switching from the WYSIWYG editor to the plain markdown editor I get the following error:

image

**Comment by wmnnd** [comment]
Thanks for catching that, I’ve fixed it now :smiley:

**Pull request merged**